.buildkite/scripts/release.sh

```diff
@@ -12,4 +12,5 @@ echo "--- Caching GPG passphrase"
 echo "$GPG_PASSPHRASE_SECRET" | gpg --armor --detach-sign --passphrase-fd 0 --pinentry-mode loopback
 
 echo "--- Release the binaries"
+export GITHUB_TOKEN="${VAULT_GITHUB_TOKEN}"
```

Comment:
I think it would be worth only doing this if `VAULT_GITHUB_TOKEN` is set so this script still works locally.

Reply:
The one problem is that I suspect `GITHUB_TOKEN` may be set as well. Otherwise the release would have complained about the token not being set.

Reply:
We could use the logic from the pre-command script:

```
if [[ "$BUILDKITE_PIPELINE_SLUG" == "terraform-provider-elasticstack-release" ]]; then
  export GITHUB_TOKEN="${VAULT_GITHUB_TOKEN}"
fi
```

Reply:
Actually I noticed now that `GITHUB_TOKEN` is set in `pre-command` as well. So I can just replace it there.